frontends/election-manager/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -95,10 +68,28 @@ export function WriteInsTranscriptionScreen({
   onClickPrevious?: () => void;
   onClose: () => void;
   onListAll: () => void;
-  saveTranscribedValue: (transcribedValue: string) => void;
+  saveTranscribedValue: (
+    ballotId: BallotId,
+    contestId: ContestId,
+    transcribedValue: string
+  ) => void;
 }): JSX.Element {
+  const previouslyTranscribedValues = [
+    'Mickey Mouse',
+    'Mickey M',
+    'Micky',
+    'Donald',
+    'Donald Duck',
+    'Roger Rabbit',
+    'RR',
+    'DD',
+    'M. Mouse',
+  ];
+
   assert(contest);
   assert(election);
+  const currentBallot = ballotsBeingAdjudicated[ballotIdxBeingAdjudicated];
+  const ballotId = currentBallot._ballotId as BallotId;
```

Comment:
I don't think we can assume a ballot ID, can we?

Reply:
Ah, I didn't realize that. So then we'll need to do something like create an `adjudicationId` for each CVR then?